src/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+
+const app = express();
+
+{
+  /* eslint-disable global-require */
+  const webpackConfig = require('../webpack.config.dev');
+  const compiler = require('webpack')(webpackConfig);
+  app.use(
+    require('webpack-dev-middleware')(compiler, { publicPath: webpackConfig.output.publicPath }),
+    require('webpack-hot-middleware')(compiler),
+  );
+  /* eslint-enable global-require */
```

Comment:
is there a reason for these requires to not be global tho?

Reply:
✨ future proofing ✨: When we support a production environment then this code block will turn into an if statement and those modules will not need to be required